beamline.rst

```diff
@@ -1,91 +1,57 @@
 Beamline API
 ============
 
-Functionality that incorporates several different instruments or
-that does not necessarily belong to one particular instrument.
+Functionality that are considered to belong to the beamlone as a whole,
+incorporating several different instruments or that does not necessarily belong
+to one particular instrument.
 
 
 List of Actuators:
 ------------------
 
-Below a list of all generic actuators across the UI (see general.rst),
-separated by data type.
-
-NB The actuators could be subdivided by component (beamline, diffractometer, etc.),
-but since the actuators can be accessed through the get_actuator(), get_actuators(),
-and set_actuator_value() top-level functions, there is no need to locate them
-within individual components. Component-specific access, such as
-
-detector.set_detector_distance(value)
-
-value = detector.get_detector_distance()
-
-could be added for convenience, but are basically superfluous,
-given that you can do
-
-set_actuator_value('detector_distance', value)
-
-value = get_actuator('detector_distance').value
+Below a list of actuators that, from an instrumentation perspective, are
+considered to belong to the beamline as a whole. Such actuators include, but
+are not limited to, actuators that have an effect on properties directly
+associated with the beamline. Such properties are for instance resolution,
+energy and resolution.
 
 
 **Actuators - Type float:**
 
     *Centring motors*
 
-        Omega, Kappa, Phi, AlignmentX,  AlignmentY,  AlignmentZ,
-        CentringX, CentringY, CentringZ,  Focus
-
-        # NB the capitalized case is to distinguish them from the current lower-case names
-             ('Phi' matches 'kappa_phi', not 'phi')
-
-        # NB There are too many motor names here, and maybe we should shorten the list.
-             It should be OK, though if the same motor is pointed to
-             by two different names. The problems is how the names match.
-             If you do not have 'CentringZ', which motor should you use during
-             centring?
-             Is 'Focus' always the same as 'AlignmentX'?
-             Does 'CentringZ' always match the same alignment motor
-             or does it depend on beamline geometry?
+        omega, kappa, kappa-phi, alignment-ver, alignment-hor, alignment-depth,
```

Comment:
Yeah, 'Phi' versus 'phi' is kind of subtle. On the other hand it would be nice to have the same axis names everywhere, and 'phi', 'kappa', 'kappa_phi' would sound rather unintuitive on a Smargon or a full kappa (or Euler) goniostat, no?
What can we come up with?

Reply:
Yes, sure that would be great for control and experiment purposes but as I mentioned above. I'm not sure that is what we need in the UI ...